src/saved-searches/ui/saved-searches-list.html

```diff
@@ -1,9 +1,9 @@
 <div class="container-fluid">
-    <h1>Mijn zoekopdrachten</h1>
+    <h1 translate-once="search.savedSearches.mySearches"></h1>
     <table class="table">
         <tr>
             <th class="saved-search-title-column">
-                <strong>Titel</strong>
+                <strong translate-once="search.savedSearches.title"></strong>
```

Comment:
Shouldn't this be `search.savedSearches.savedSearches` ?

Reply:
No it's literally 'Titel' and 'Titre' as a new key 'title', that's why I changed the other key